ddtrace/llmobs/_experiment.py

```diff
@@ -28,3 +29,35 @@ def __init__(self, name: str, dataset_id: str, data: List[DatasetRecord]) -> Non
         self.name = name
         self._id = dataset_id
         self._data = data
+
+
+class Experiment:
+    def __init__(
+        self,
+        name: str,
+        task: Callable[[Dict[str, Any]], Any],
```

Comment:
```suggestion
        task: Callable[[Dict[str, JSONType]], Any],
```
right? since we have to be able to encode the input data at some point?